samples/snippets/src/test/java/com/example/dialogflow/CreateConversationProfileTest.java

```diff
@@ -40,21 +41,38 @@
 public class CreateConversationProfileTest {
 
   private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+  private static final String LOCATION = "global";
+  private static String conversationProfileNameToDelete = "";
```

Comment:
`null` is probably a better choice for empty than empty string

Reply:
Yes thanks! Done.